app/models.py

```diff
@@ -750,32 +754,71 @@ def pending_submission_reminder(cls, session: Session) -> "Query[Self]":
             col(cls.id).notin_(Message.application_by_type(MessageType.BORROWER_PENDING_SUBMIT_REMINDER)),
         )
 
+    @classmethod
+    def pending_external_onboarding_reminder(cls, session: Session) -> "Query[Self]":
```

Comment:
Correct, but hopefully for new submissions they have already accessed the external onboarding system by then (as now as soon as they send the application both, the front end and the email confirmation ask for accessing the onboarding system)

Reply:
It would be better to have a consistent amount of time between when they are initially provided the onboarding link, and when they are reminded about the onboarding link.

As is, they will always be reminded within 24 hours. I think the reminder will be more effective if it occurs after a reasonable delta.

Reply:
Also, method right now is very sensitive to the frequency of the send-reminders cron job. If we ran the reminders more frequently, it shouldn't change the behavior of the app.